__tests__/app.test.js

```diff
@@ -89,3 +103,37 @@ describe('GET /api/articles/:article_id', () => {
     });
   });
 });
+
+describe('GET /api/articles', () => {
+  
+  test('should respond with status 200 and array of objects of the articles', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({ body: { articles }}) => {
+      articles.forEach((article) => {
+        expect(article).toHaveProperty('author')
+        expect(article).toHaveProperty('title')
+        expect(article).toHaveProperty('article_id')
+        expect(article).toHaveProperty('topic')
+        expect(article).toHaveProperty('created_at')
+        expect(article).toHaveProperty('votes')
+        expect(article).toHaveProperty('article_img_url')
+        expect(article).toHaveProperty('comment_count')
+
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(String)
```

Comment:
Here, on the other hand, you've done perfectly well. Because we are checking all articles with a for each, we wouldn't expect to want any specifics on the keys and their properties. However, we know how many articles we're expecting to get back, so we should definitely check for the amount of articles we're getting back.